src/utils/AdbWrapper.js

```diff
@@ -862,11 +862,16 @@ export default class AdbWrapper {
     output = await this.executeCommand("rm -r /opt");
     output = await this.executeCommand("mount -o ro,remount /");
 
+    let retries = 0;
     output = await this.executeCommand(`rm -r ${this.wtfos.path}`);
-    if(output.exitCode !== 0) {
-      statusCallback("ERROR: Failed cleaning up");
-      output.stdout.split("\n").forEach((line) => statusCallback(line));
-      return;
+    while(output.exitCode !== 0) {
```

Comment:
I think using a do here would be more readable and executeCOmmand would only need to be called in `do`